src/Invoice/Asset/invoice/css/components.css

```
@@ -502,6 +504,65 @@ textarea.form-control {
+    position: relative;
+    overflow: hidden;
+}
+
+/* ColumnResizer.autoFit() deliberately fits a column's width to its *data*,
+   not its header label — a long header on a mostly-empty column (e.g.
+   "Street Address (cont'd)") shouldn't itself force the column wide. Letting
+   the label wrap onto a second line, instead of forcing single-line width,
+   is what makes that possible. */
+.resizable-grid thead th {
+    white-space: normal;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)